elasticsearch/templates/elasticsearch.yml.j2

```diff
@@ -0,0 +1,119 @@
+# ======================== Elasticsearch Configuration =========================
```

Comment:
use lineinfile if changes are specific

Reply:
I tried with lineinfile first but due to multiple changes it was throwing multiple errors. I'll do it next time.